libs/geospatial/src/test/java/org/codice/ddf/libs/geo/GeoUtilTest.java

```diff
@@ -266,18 +266,25 @@ public void testConvertPointRadiusToCirclePolygon() throws ParseException {
     double lat = 43.25;
     double lon = -123.45;
     double radius = 100;
+    double tolerance = 0.000001; // difference of less than a meter
```

Comment:
❓ Did you confirm that this is less than a meter? I might have been off by an order of magnitude when I mentioned it earlier.

Reply:
Yes, 6 decimal places is < 1 meter of precision.

https://en.wikipedia.org/wiki/Decimal_degrees